README.md

```diff
@@ -5,6 +5,7 @@
 # Cognite Python Replicator
 [![build](https://webhooks.dev.cognite.ai/build/buildStatus/icon?job=github-builds/cognite-replicator/master)](https://jenkins.cognite.ai/job/github-builds/job/cognite-replicator/job/master/)
 [![codecov](https://codecov.io/gh/cognitedata/cognite-replicator/branch/master/graph/badge.svg)](https://codecov.io/gh/cognitedata/cognite-replicator)
+[![Documentation Status](https://readthedocs.com/projects/cognite-replicator/badge/?version=latest)](https://cognite-cognite-replicator.readthedocs-hosted.com/en/latest/)
```

Comment:
https://cognite-cognite-replicator.readthedocs-hosted.com/en/latest/ is this the correct link?